src/MySql/Clause/DuplicateKeyClause.php

```diff
@@ -8,10 +8,7 @@
 
 trait DuplicateKeyClause
 {
-    /**
-     * @var AssignmentExpression
-     */
-    protected $assignmentOnUpdate;
+    protected ?AssignmentExpression $assignmentOnUpdate = null;
```

Comment:
Here, I have intentionally omitted the type to be able to more flexibly replace the expression type in clauses.